resource-timing/buffer-full-add-after-full-event.html

```diff
@@ -72,6 +72,9 @@
     await loadRandomResource();
     await waitForEventToFire();
     await clearAndAddAnotherEntryToBuffer();
+    // Since we have no strict guarantees when an entry will be added to the
+    // buffer, waiting till next task to try to avoid flakiness.
+    await waitTillNextTask();
```

Comment:
s/waitTillNextTask/waitForNextTask

Reply:
nm. I misunderstood your comment and missed my own typo. Fixing